src/routes/README.md

```diff
@@ -0,0 +1,27 @@
+# Routes Folder
+
+Routes define routes within your application. Fastify provides an
+easy path to a microservice architecture, in the future you might want
+to independently deploy some of those.
```

Comment:
Highlighting concepts like `modular monolith` can be of help here

Reply:
Yep, a lot of text is auto-generated by fastify-cli, I think I am gonna improve it after this demo.

Reply:
https://github.com/fastify/fastify-cli/pull/736